### DISCUSSION TITLE
How to add a button after the Cateogry name

### DISCUSSION THREAD

**Discussion by MrNikolovski** [GrapesJS/grapesjs#4221] [Q&A]
Hi all,

We were able to add an HTML code in the Category name and it was rending the HTML.  This was working on v0.16.30
![image](https://user-images.githubusercontent.com/12418977/160552115-1af5e3a6-6000-48ca-9853-a64ccd772bf2.png)

After updating to the latest, this is not working anymore and the HTML code is considered a string. Is there a way to implement this?

Thank you.

**Comment by artf** [comment]
You should probably use a custom block manager https://grapesjs.com/docs/modules/Blocks.html#customization
